imports/plugins/core/shipping/client/templates/settings/shipping.js

```diff
@@ -48,8 +48,6 @@ Template.shippingSettings.events({
     if (packageId) {
       // update package registry
       Meteor.call("registry/update", packageId, settingsKey, fields);
-      // also update shipping provider status
-      Meteor.call("shipping/provider/toggle", packageId, settingsKey);
```

Comment:
Are we performing this functionality in some other way now?

Reply:
I didn't see where this was being used. I believe it was only going by the package being enabled, so the first Meteor.call is the only one actually having an effect.